web/.eslintrc.js

```diff
@@ -0,0 +1,104 @@
+/*
+ reference: https://github.com/toshi-toma/eslint-config-airbnb-typescript-prettier
+*/
+module.exports = {
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    sourceType: "module",
+    project: "./tsconfig.json",
+    tsconfigRootDir: "./",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  env: {
+    browser: true,
+    es2017: true,
+  },
+  extends: [
+    "airbnb",
+    "airbnb/hooks",
+    "prettier",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:import/typescript",
+  ],
+  plugins: ["react", "jsx-a11y", "import", "prettier", "@typescript-eslint", "simple-import-sort",],
+  ignorePatterns: [".eslintrc.js", "webpack.config.js"],
+  globals: {},
+  rules: {
+    // prettier
+    "prettier/prettier": ["error"],
+
+    // TypeScript
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/ban-ts-ignore": "off",
+    "@typescript-eslint/no-unused-vars": "error",
+    "@typescript-eslint/explicit-member-accessibility": "off",
+    "@typescript-eslint/no-object-literal-type-assertion": "off",
+    "@typescript-eslint/no-shadow": "warn",
+    "@typescript-eslint/ban-types": "warn",
+    "@typescript-eslint/no-explicit-any": "warn",
+    /* 명시적인 return type 작성*/
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+
+    // js
+    "camelcase": "off",
+    "react/require-default-props": "off",
+    "consistent-return": "off",
```

Comment:
p3; 요정도 타협... 불가능할까요?ㅎㅎㅎ

```suggestion
    "consistent-return": "off",
    "no-console": ["warn", { "allow": ["error", "warn"] }],
```

Reply:
^ 저는 좋은 것 같습니다. 
반영하겠습니다. @hajoeun

Reply:
https://github.com/banksalad/styleguide/pull/35/commits/5a3563167a863adbb1e8e90feed1b47de579f5af
반영완료